stargate-core/app/com/salesforce/mce/stargate/controllers/McSsoController.scala

```diff
@@ -64,6 +64,7 @@ class McSsoController @Inject() (
     val result = Ok.withCookies(Cookie(
       name = sessionConfig.cookieName,
       value = "",
+      maxAge = Some(0),   // ensure cookie expires immediately, if not discarded
```

Comment:
why do we need this? I thought it already clears the cookie value, what's the point of setting maxAge

Reply:
I created an issue and documented. It won't discard the cookie but will retain a cookie with empty value, like: 

`PLAY_SESSION=; Path=/; Domain=localhost; Secure; HttpOnly;`

This seems to be a known issue with play. The effect is that this is generating thousands of error logs with the message `Cookie failed message authentication check `

A simple local test confirmed that adding maxAge will resolve the issue.